### ISSUE TITLE
Update FAQ for canceling

### ISSUE THREAD

**Issue by tarakieffer** [popuparchive/pop-up-archive#1204]
Community plan is called '1 Hour Demo.'

![screen shot 2015-08-20 at 9 30 26 am](https://cloud.githubusercontent.com/assets/9256525/9385841/39d9db7e-471e-11e5-984e-3c45911592d1.png)

Also, just to confirm, is it okay that a paid user who downgrades to the 1 Hour Demo (and has not previously been on the 1 Hour Demo) gets the free hour?

**Comment by Saltzshaker** [comment]
Thanks @tarakieffer. I think it's ok for them to still get the free hour, if they haven't used it previously. @annewootton @baileyspace if you feel differently, feel free to open a separate ticket.

Also, just for _me_ to confirm, we're no longer doing any pro-rating, so I should delete that bit as well?

**Comment by pkarman** [comment]
yes, the free demo hour is lifetime of the User.

**Comment by annewootton** [comment]
Yes to lifetime free hour, yes to removing proration mention. Good to deploy once FAQ changes are in.

**Comment by Saltzshaker** [comment]
https://github.com/popuparchive/pop-up-archive/commit/3f5767c515d76e0cb281e5b43841e46cb6e6a849
